src/battle_util.c

```diff
@@ -10991,15 +10991,22 @@ void SortBattlersBySpeed(u8 *battlers, bool32 slowToFast)
 void TryRestoreHeldItems(void)
 {
     u32 i;
-    u16 lostItem = ITEM_NONE;
-
+    bool32 returnNPCItems = B_RETURN_STOLEN_NPC_ITEMS >= GEN_5 && gBattleTypeFlags & BATTLE_TYPE_TRAINER;
```

Comment:
We don't tend to use variables like these in my experience. I get wanting to streamline it, but I think we'd sooner use a define rather than a variable for it?

Reply:
I put this logic into a #define above the function called RETURN_NPC_ITEMS.

Reply:
Upon further discussion with other parties, we decided to revert it back to a ``bool32.``